includes/admin/services/class-rop-linkedin-service.php

```diff
@@ -591,10 +595,38 @@ public function share( $post_details, $args = array() ) {
 			return false;
 		}
 
-		$body = json_decode( wp_remote_retrieve_body( $response ), true );
+		$body    = json_decode( wp_remote_retrieve_body( $response ), true );
+		$headers = wp_remote_retrieve_headers( $response );
 
 		if ( null === $body ) {
 
+			if ( ! empty( $this->share_link_text ) ) {
+				$post_id = isset( $headers['x-restli-id'] ) ? $headers['x-restli-id'] : '';
+				if ( ! class_exists( 'ROP_Pro_Linkdin_Helper' ) ) {
+					include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+
+					if ( is_plugin_active( 'tweet-old-post-pro/tweet-old-post-pro.php' ) ) {
+						require_once ROP_PRO_PATH . 'includes/helpers/class-rop-pro-linkdin-helper.php';
+					}
+				}
+
+				if ( class_exists( 'ROP_Pro_Linkdin_Helper' ) ) {
```

Comment:
The same thing here. PRO plugin should be able to load its classes.

Reply:
@Soare-Robert-Daniel Done!